app/uk/gov/hmrc/tradergoodsprofiles/utils/ValidationSupport.scala

```diff
@@ -22,7 +22,10 @@ import uk.gov.hmrc.tradergoodsprofiles.models.requests.UpdateRecordRequest
 import uk.gov.hmrc.tradergoodsprofiles.services.UuidService
 import uk.gov.hmrc.tradergoodsprofiles.utils.ApplicationConstants._
 
+import java.util.UUID
 import scala.reflect.runtime.universe.{TypeTag, typeOf}
+import scala.util.{Failure, Success, Try}
```

Comment:
can you remove the unused import Line 25,27

Reply:
Removed